.devops/deploy-website.yml

```diff
@@ -1,5 +1,10 @@
 trigger:
-- develop
+  branches:
+    include:
+    - develop
```

Comment:
main??

Reply:
was develop before... just added paths so that this build runs only when something changed in the website folder. anyway GitHub actions do not support paths in combination with azure pipelines